include/net/aws_iot.h

```diff
@@ -98,6 +98,8 @@ enum aws_iot_evt_type {
 	AWS_IOT_EVT_FOTA_ERASE_DONE,
 	/** FOTA progress notification. */
 	AWS_IOT_EVT_FOTA_DL_PROGRESS,
+	/** FOTA error. */
```

Comment:
```suggestion
	/** FOTA error. Used to propagate FOTA-related errors to the application. This is to distinguish between AWS_IOT irrecoverable errors and FOTA errors, so they can be handled differently. */
```

It doesn't hurt to explain *why* this "FOTA error" event exists.

Reply:
Ok, this works. Ill add the suggestion. In general I agree that we should be very clear in the way we document code, but there is always the struggle of pinpointing whats obvious and whats not.

Reply:
![image](https://user-images.githubusercontent.com/188915/108385150-abf37d00-720b-11eb-9f85-818f1faba439.png)

IMHO "FOTA error." provides exactly zero meaningful documentation. Because these two words are already included in the symbol name: AWS_IOT_EVT_**FOTA**_**ERROR**.